src/main/kotlin/kscript/app/code/GradleTemplates.kt

```diff
@@ -39,53 +39,114 @@ object GradleTemplates {
         """.trimIndent()
     }
 
-    //Capsule: https://github.com/ngyewch/gradle-capsule-plugin
+    fun hangingIndents(s: String, n: Int): String {
```

Comment:
`trimMargin()` trims margins. This function _adds_ a margin of _n_ spaces on all except the first line. This is so that multi-line dependencies look nice in the output. It's purely cosmetic, but with it, the `dependencies` section in the package's `build.gradle.kts` looks something like this:
```
dependencies {
    implementation(files("/home/vinay/.kscript/cache/jar_db71153912c566b4d192ee7f36d85a8a/scriplet.jar"))
    implementation("org.jetbrains.kotlin:kotlin-stdlib")
    implementation("org.jetbrains.kotlin:kotlin-script-runtime:1.6.21")
    implementation("com.github.ajalt.clikt:clikt:3.4.2")
}
```
Without it, you get
```
        dependencies {
            implementation(files("/home/vinay/.kscript/cache/jar_cfdf0ec49939ebcc9ca0b9a9cbe98f4c/scriplet.jar"))
            implementation("org.jetbrains.kotlin:kotlin-stdlib")
implementation("org.jetbrains.kotlin:kotlin-script-runtime:1.6.21")
implementation("com.github.ajalt.clikt:clikt:3.4.2")
        }
```
So functionally the same, but looks uglier.